README.md

```diff
@@ -119,7 +119,7 @@ cat - > /tmp/scep.csr
 $ ./scepclient-linux-amd64 -help
 Usage of ./scepclient-linux-amd64:
   -ca-fingerprint string
-    	md5 fingerprint of CA certificate for NDES server.
+    	(Note: Changed from MD5) SHA-256 digest of CA certificate for NDES server.
```

Comment:
nit: see note below

Reply:
Also, thanks for remember to update the docs! :)

Reply:
Done. thanks :)